src/server/controllers/collectives.js

```diff
@@ -28,3 +29,86 @@ export async function info(req, res, next) {
 
   res.send(response);
 }
+
+const query = `
+query Account($slug: String) {
+  account(slug: $slug) {
+    memberOf(role: HOST) {
+      nodes {
+        createdAt
+        account {
+          ... on Collective {
+            slug
+            name
+            website
+            description
+            isApproved
+            balance
```

Comment:
Should not use that field. But stats { balance { value } }. You will also find the yearlyBudget in stats.

Was merged yesterday, not sure if it's in prod yet.

Reply:
Okay. Thanks